cobalt/app/cobalt_switch_defaults_starboard.cc

```diff
@@ -111,6 +111,8 @@ const base::CommandLine::SwitchMap GetCobaltParamSwitchDefaults() {
         // Enable precise memory info so we can make accurate client-side
         // measurements.
         {switches::kEnableBlinkFeatures, "PreciseMemoryInfo"},
+        // Enable autoplay video/audio.
```

Comment:
micro-nit: I'd add also the "why" here, e.g.
`// Enable autoplay video/audio, otherwise loaded videos don't play 
automatically upon load.` Or some such.

Reply:
If we're going to add a comment, I think it should be more telling of why we need it and say `We may launch directly into media playback before user interaction`